### DISCUSSION TITLE
Missing lengths for purl parts / and purl as a whole

### DISCUSSION THREAD

**Discussion by xSeagullx** [package-url/purl-spec#493] [Ideas]
Hey!

I did not find anything about the max length of purl, which is quite important, for example if you need that purl to be stored in relational databases.

**Comment by pombredanne** [comment]
@xSeagullx as explained in https://github.com/package-url/purl-spec/issues/289#issuecomment-3009191432 there is no length we can set for purl parts (aka. components). Some ecosystems may have their own limits that we could specify in a purl type definition for reference, but unless someone can come with the list of limits for a type, (and preferably all types) there is nothing we can do much.

I am therefore moving this to a discussion so we can explore this further.
